src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -0,0 +1,324 @@
+        }
+
+        $title = $this->slackBlocksBuilder->buildSimpleSection(sprintf('*Title :* %s', $submit->getTalk()->getTitle()));
+
+        $text = \count($submit->getUsers()) > 1 ? '*Speakers*' : '*Speaker*';
+        $text .= ' : '.$submit->reduceSpeakersNames();
+        $text .= ' *Conference* : ';
+        $text .= sprintf('<%s|%s>', $submit->getConference()->getSiteUrl(), $submit->getConference()->getName());
+        $textBlock = $this->slackBlocksBuilder->buildSimpleSection($text);
+
+        $talkUrl = $this->router->generate('easyadmin', [
+            'id' => $submit->getTalk()->getId(),
+            'entity' => 'Talk',
+            'action' => 'show',
+        ], UrlGeneratorInterface::ABSOLUTE_URL);
+
+        $context = $this->slackBlocksBuilder->buildContext(sprintf('<%s|See the talk on Starfleet>', $talkUrl));
+
+        $blocks = [
+            'blocks' => [
+                $header,
+                $this->slackBlocksBuilder->buildDivider(),
+                $title,
+                $textBlock,
+                $this->slackBlocksBuilder->buildDivider(),
+                $context,
+            ],
+        ];
+
+        $this->notify($blocks);
+    }
+
+    /**
+     * @param array<Conference> $conferences
+     *
+     * @return array<array>
+     */
+    private function buildConferencesBlocks(array $conferences): array
+    {
+        $header = $this->slackBlocksBuilder->buildHeader('New conferences of the day');
+
+        if (0 === \count($conferences)) {
+            $conferencesBlocks = $this->slackBlocksBuilder->buildSimpleSection('No conferences were added today !');
+        } else {
+            $conferencesBlocks = [];
+
+            foreach ($conferences as $conference) {
+                if (null === $conference->getCfpUrl()) {
+                    continue;
+                }
+
+                if ($conference->getExcluded()) {
+                    continue;
+                }
+
+                $conferenceUrl = $this->router->generate('easyadmin', [
+                    'id' => $conference->getId(),
+                    'entity' => 'NextConference',
+                    'action' => 'show',
+                ], UrlGeneratorInterface::ABSOLUTE_URL);
+
+                $location = $conference->isOnline() ? 'Online' : sprintf(':flag-%s:', $conference->getCountry());
+
+                $conferenceText = sprintf('*<%s|%s>*, %s (<%s|Go to CFP>)', $conferenceUrl, $conference->getName(), $location, $conference->getCfpUrl());
+                $conferencesBlocks[] = $this->slackBlocksBuilder->buildSectionWithButton($conferenceText, 'Mute this conference', 'Mute Conference', $conference->getId());
+            }
+        }
+
+        // This looks really ugly but if we add only one conference, it can't be unpacked by the splat operator beause it is a string array
+        // Will be fixed with PHP8
+        if (\array_key_exists('type', $conferencesBlocks)) {
```

Comment:
This is weird, because the conference is IN an array, so we never met this condition, isn't?

Anyway, you can use array_merge instead of this hack